operator/controllers/external_scaler.go

```diff
@@ -47,12 +50,13 @@ func createExternalScaler(
 	)
 	logger.Info("Creating external scaler Deployment", "Deployment", *scalerDeployment)
 	deploymentsCl := cl.AppsV1().Deployments(appInfo.Namespace)
-	if _, err := deploymentsCl.Create(scalerDeployment); err != nil {
+	if _, err := deploymentsCl.Create(ctx, scalerDeployment, metav1.CreateOptions{}); err != nil {
```

Comment:
I've just quickly looked at the code (haven't got time to do proper review), but why don't you use the controller-runtime client, why should be already set up in the controller?

Reply:
Moved to #47 due to DCO.

I never actually thought of changing it since we were following a baseline, but now that you mentioned, I'll review the code to make sure no other clients are being used!